### ISSUE TITLE
Terms and conditions link at VERY bottom of API contest page

### ISSUE THREAD

**Issue by tscheuing** [itglue/api-contest#17]
Note that the terms and conditions for submission into the contest is at the very bottom of the contest page, below a significant amount of white space. I would strongly advise reading this and understanding what you agree to give up by submitting to this contest. 

ITG should have made this link significantly more noticeable, in my opinion.

**Comment by firefox15** [comment]
Completely agree.  We were excited when this contest was announced because our company has spent a significant amount of time automating the collection of a ton of data for import into IT Glue.  We thought we had a good shot at winning.  Then we read the terms and conditions (which, like you said, we really had to search for):

> 6.2 **You grant to us an exclusive, worldwide, royalty-free, perpetual and irrevocable licence to copy, store, edit, distribute, transmit and publish your competition entries.**

> 6.5 To the maximum extent permitted by applicable law, **you irrevocably and unconditionally waive your rights to be identified as the author of your competition entries and to object to any derogatory treatment of your competition entries.**

> 7.1 Competition entries will not be returned. Please, therefore, keep a copy of your competition entries. Entrants agree that **all submissions** verified by IT Glue’s Development team will appear on the IT Glue GitHub, and **may be used or adapted by IT Glue**. The competition winner agrees that their entry will appear on the IT Glue website.  

Given the work we have put into automation, we would _never_ agree to such conditions.  It's pretty clear that this contest exists so that IT Glue can repackage good-looking solutions into the product, sell them as their own, and not even have to reference the original author of it.  These T&Cs apply regardless if you win or not.  Our company held a meeting, and we unanimously agreed not to enter.

I hope all those who enter realize what they are giving up for a chance at a gift card.  I have a feeling our company's decision was not unique, and I wonder how many more entries they would have received if they didn't try to take possession of everyone's work.

**Comment by tscheuing** [comment]
Exactly. Basically, we have to submit only our own work, but we relinquish any rights or claim to it once submitted and ITG can do anything they want to it, royalty-free. Seems like an abuse of trust, especially given the terms were not as obvious as they should have been.
